.github/workflows/pr_title_checker.yml

```diff
@@ -0,0 +1,18 @@
+name: "PR Title Checker"
```

Comment:
Can we change the title to `PR Title matches regex "^(Revert \")?WT-[0-9]+ "`?
Right now when the check fails there's no obvious way to know what's gone wrong. Having this in the check name makes it obvious